docs/versionhistory.rst

```diff
@@ -37,6 +37,8 @@ This library adheres to `Semantic Versioning 2.0 <http://semver.org/>`_.
 - Fixed quitting the debugger in a pytest test session while in an active task group
   failing the test instead of exiting the test session (because the exit exception
   arrives in an exception group)
+- Re-add support for linux abstract namespace sockets (#781
```

Comment:
What did you mean by "Re-add" (also notice the tense which differs from all the rest of the changelog entries)? Did AnyIO used to have this support before?

Reply:
Indeed. It worked/works in 3.7.1 which we use currently to delay the ExceptionGroup change. I havn't bisected or tried to find where it broke more closely yet though.

Reply:
If this used to work before, then I guess then the correct phrasing would be "fixed" because I never intentionally removed support for it.